docs/advanced.rst

```diff
@@ -1018,105 +1136,665 @@ to learn more about the use of this feature.
++------------------------------------------------------------+
+| .. literalinclude:: ../code/XMLTester.xml                  |
+|    :language: xml                                          |
+|    :start-after: <!-->CONF-SERVER-SERVER-PREFIX<-->        |
+|    :end-before: <!--><-->                                  |
++------------------------------------------------------------+
+
+Note that a *server* can act as a *client* of other *servers*.
+Thus, the following section may also apply.
+
+Client side setup
+""""""""""""""""""
+
+Each *client* must keep a list of the *servers* it wants to link to.
+Each single element represents an individual server and a ``GuidPrefix`` must be provided.
+The *server* list is the attribute:
+
+.. code-block:: bash
+
+    ParticipantAttributes.rtps.builtin.discovery_config.m_DiscoveryServers
+
+and must be populated with ``RemoteServerAttributes`` objects with a valid ``guidPrefix`` member.
+In xml the server list is and its elements are simultaneously specified.
+Note that ``prefix`` is an attribute of the ``RemoteServer`` tag.
+
+.. code-block:: bash
+
+    dds>profiles>participant>rtps>builtin>discovery_config>discoveryServerList>RemoteServer@prefix
+
++------------------------------------------------------------+
+| **C++**                                                    |
++------------------------------------------------------------+
+| .. literalinclude:: ../code/CodeTester.cpp                 |
+|    :language: c++                                          |
+|    :start-after: //CONF_SERVER_CLIENT_GUIDPREFIX           |
+|    :end-before: //!--                                      |
++------------------------------------------------------------+
+| **XML**                                                    |
++------------------------------------------------------------+
+| .. literalinclude:: ../code/XMLTester.xml                  |
+|    :language: xml                                          |
+|    :start-after: <!-->CONF-SERVER-CLIENT-PREFIX<-->        |
+|    :end-before: <!--><-->                                  |
++------------------------------------------------------------+
+
+.. _DS_locators:
+
+The server locator list
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Each *server* must specify how to be reach by specifying valid locators.
+Any *client* must be given proper locators to reach each of its *servers*.
+As in the :ref:`above section <DS_guidPrefx>` here there is a *server* and a *client* side setup.
+
+Server side setup
+"""""""""""""""""
+
+In the examples below its shown how to setup the locator list attribute (note that discovery strategy only deals with
+metatraffic attributes):
+
+.. code-block:: bash
+
+    ParticipantAttributes.rtps.builtin.(metatrafficMulticastLocatorList|metatrafficUnicastLocatorList)
+
+and xml tag:
+
+.. code-block:: bash
+
+    dds>profiles>participant>rtps>builtin>(metatrafficMulticastLocatorList|metatrafficUnicastLocatorList)
+
```

Comment:
Get rid of the enum and tag hierarchy, as they are implicit on the examples.

Reply:
As above... It's not easy to imply the jigsaw puzzle that the attributes are, directly, from the examples...that's the main reason I put these clues there.

Reply:
Same changes as above